src/travis2docker/templates/build.sh

```diff
@@ -168,6 +168,108 @@ chown_all(){
     chown -R odoo:odoo /etc/postgresql-common/common-vauxoo.conf
 }
 
+configure_vim(){
+    if [ -z ${VIM_INSTALL+x} ];
+    then
+      return 0;
+    fi
+    # Upgrade & configure vim
+    apt install vim --only-upgrade
+    # Get vim version
+    VIM_VERSION=$(dpkg -s vim | grep Version | sed -n 's/.*\([0-9]\+\.[0-9]\+\)\..*/\1/p' | sed -r 's/\.//g')
+
+    wget -q -O /usr/share/vim/vim${VIM_VERSION}/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
+    git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"
+    git_clone_copy "${VIM_OPENERP_REPO}" "master" "vim/" "${HOME}/.vim/bundle/vim-openerp"
+    git_clone_copy "${VIM_JEDI_REPO}" "master" "." "${HOME}/.vim/bundle/jedi-vim"
+
+    sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
+    sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
+
+    # Disable virtualenv in Pymode
+    cat >> ~/.vimrc << EOF
+" Disable virtualenv in Pymode
+let g:pymode_virtualenv = 0
+" Disable pymode init and lint because of https://github.com/python-mode/python-mode/issues/897
+let g:pymode_init = 0
+let g:pymode_lint = 0
+" Disable modelines: https://github.com/numirias/security/blob/master/doc/2019-06-04_ace-vim-neovim.md
+set modelines=0
+set nomodeline
+EOF
+
+    # Disable vim-signify
+    cat >> ~/.vimrc << EOF
+" Disable vim-signify
+let g:signify_disable_by_default = 1
+EOF
+
+    # Install and configure YouCompleteMe
+    VIM_YOUCOMPLETEME_PATH="${HOME}/.vim/bundle/YouCompleteMe"
+    git clone ${VIM_YOUCOMPLETEME_REPO} ${VIM_YOUCOMPLETEME_PATH}
+    # Install the custom version of YouCompleteMe because the last required g++ 4.9
+    (cd "${VIM_YOUCOMPLETEME_PATH}" && git reset --hard c31152d34591f3211799ca1fe918eb78487e6dde && git submodule update --init --recursive && ./install.py)
+    cat >> ~/.vimrc << EOF
+" Disable auto trigger for youcompleteme
+let g:ycm_auto_trigger = 0
+EOF
+
+    # Install WakaTime
+    git_clone_copy "${VIM_WAKATIME_REPO}" "master" "." "${HOME}/.vim/bundle/vim-wakatime"
+
+    cat >> ~/.vimrc << EOF
+colorscheme heliotrope
+set colorcolumn=119
+set spelllang=en,es
+EOF
+
+    # Configure pylint_odoo plugin and the .conf file
+    # to enable python pylint_odoo checks and eslint checks into the vim editor.
+    cat >> ~/.vimrc << EOF
+:filetype on
+let g:syntastic_aggregate_errors = 1
+let g:syntastic_python_checkers = ['pylint', 'flake8']
+let g:syntastic_auto_loc_list = 1
+let g:syntastic_python_pylint_args =
+    \ '--rcfile=/.repo_requirements/linit_hook/travis/cfg/travis_run_pylint_vim.cfg --valid_odoo_versions=14.0'
+let g:syntastic_python_flake8_args =
+    \ '--config=/.repo_requirements/linit_hook/travis/cfg/travis_run_flake8.cfg'
+let g:syntastic_javascript_checkers = ['eslint']
+let g:syntastic_javascript_eslint_args =
+    \ '--config /.repo_requirements/linit_hook/travis/cfg/.jslintrc'
```

Comment:
This path doesn't exists anymore but you can use the same than pre-commit-vauxoo